mlfinlab/feature_importance/fingerpint.py

```diff
@@ -0,0 +1,181 @@
+"""
+Implements model fingerprint algorithm from 'Beyond the Black Box' paper
```

Comment:
Please Mention full name of paper and full names of authors.

Also add a description of why we added this, whats its value add, lets build a bit of a story.